appveyor.yml

```diff
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
```

Comment:
```suggestion
  - julia_version: 1.3  # Latest release
```

Reply:
A.V. Actually lets us specify that as `1`
https://github.com/JuliaCI/Appveyor.jl